internal/operators/cnv/config.go

```diff
@@ -13,6 +13,10 @@ type Config struct {
 	SupportedSRIOVNetworkIC DeviceIDDecoder `envconfig:"CNV_SUPPORTED_SRIOV_NICS" default:"8086:158b,15b3:1015,15b3:1017,15b3:1013,15b3:101b"`
 	// CNV operator mode. It defines whether to use upstream `false` or downstream `true`
 	Mode bool `envconfig:"CNV_MODE" default:"true"`
+	// HPP install flag. It defines whether we should install HPP when CNV is on SNO
+	SNOInstallHPP bool `envconfig:"CNV_SNO_INSTALL_HPP" default:"true"`
```

Comment:
Please update documentation so people know how to use it and add support in https://github.com/openshift/assisted-test-infra.

Reply:
Is there a doc of these I am missing?

Reply:
Please add a section how to enable hpp [here](https://github.com/openshift/assisted-service/blob/140fe0cc2726d73077e6737a8a65f178d2ea6875/docs/dev/olm-operator-plugins.md)

Reply:
and add the env var to https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml

Reply:
> and add the env var to https://github.com/openshift/assisted-service/blob/master/openshift/template.yaml

Added, defaults to `false` until 4.10 testing begins, does that make sense? I have tested this against a hacked assisted-test-infra environment with 4.10 OC & CNV, but I understand we are still not testing those (waiting for FC?)

> Please add a section how to enable hpp [here](https://github.com/openshift/assisted-service/blob/140fe0cc2726d73077e6737a8a65f178d2ea6875/docs/dev/olm-operator-plugins.md)

No problem, I just want to make sure this is the correct place for it - it seems like an OLM operator addition development doc

Reply:
Setting as default is fine, but if we want to enable it in the cloud i prefer not get another release, so adding it to the template will enable us just to change app-sre yaml and don't touch the code. 
As far as i know we didn't start to test 4.10, @osherdp correct me if i'm wrong

Reply:
no, we haven't integrated 4.10 yet
still waiting or a fc payload